bitbots_docs/docs/manual/testing/test_robot_hardware.rst

```diff
@@ -27,32 +27,31 @@ Manual procedure
-        - start on your laptop ``roslaunch bitbots_ros_control viz_pressure.launch``
+    motor power on and ``ros2 launch bitbots_ros_control ros_control_standalone.launch only_pressure:=true``
+        - start on your laptop ``ros2 launch bitbots_ros_control viz_pressure.launch``
             - you should see the pressure values as arrows in rviz as well as the center of pressures
             - press on the sensors to see if they behave correctly
-        - maybe use plottjuggler to get more details and see eventual drift
+        - maybe use plotjuggler to get more details and see eventual drift
 
 #. Test servos
-    motor power off and ``roslaunch bitbots_ros_control ros_control_standalone.launch torqueless_mode:=true``
-        - it should give you errors for not being able to ping servos
-    motor power on and ``roslaunch bitbots_ros_control ros_control_standalone.launch torqueless_mode:=true``
+    motor power off and ``ros2 launch bitbots_ros_control ros_control_standalone.launch torqueless_mode:=true``
+        - it should give you an error because the motor power is off
+    motor power on and ``ros2 launch bitbots_ros_control ros_control_standalone.launch torqueless_mode:=true``
         - it should start without any errors
         - servos should be torqueless (not stiff)
-        - start on your laptop ``roslaunch bitbots_ros_control viz_servos.launch`` you should see the the robot, the TF tree and the efforts
-            - move the robot around to see if it behaves correctly (ignore efforts for now, since we are torqueless)
-            - start rqt diagnostic viewer, you should see all servos on green and get voltage, temperature and error status
+        - start on your laptop ``ros2 launch bitbots_ros_control viz_servos.launch`` you should see the the robot, the TF tree and the efforts
+            - move the robot around to see if it behaves correctly
+            - start rqt robot monitor, check "Alternative view" and you should see all servos on OK
+            - start the runtime monitor in rqt and you can get voltage, temperature and error status
             - maybe use plotjuggler to see the values in more detail
 
-    turn motor power on and ``roslaunch bitbots_ros_control ros_control_standalone.launch``
-        - it should start whitout any errors
+    turn motor power on and ``ros2 launch bitbots_ros_control ros_control_standalone.launch``
+        - it should start without any errors
         - servos should be half stiff, but still moveable
-        - start on your laptop ``roslaunch bitbots_ros_control viz_servos.launch`` you should see the the robot, the TF tree and the efforts
-            - run ``rosrun bitbots_ros_control send_joint_command.py``, the robot should go into init pose and be completly stiff
-            - apply some force on a servo, the effort visualization should show it (if not maybe set read_effort via dynamic reconfigure to active)
```

Comment:
What's with this line?

Reply:
The effort visualization plugin is not migrated to ROS2 yet, I will add the line back once that's done